style/properties/shorthands/text.mako.rs

```diff
@@ -69,21 +65,26 @@
         #[allow(unused)]
         fn to_css<W>(&self, dest: &mut CssWriter<W>) -> fmt::Result where W: fmt::Write {
             use crate::values::specified::TextDecorationLine;
+            use crate::values::specified::Color;
```

Comment:
You could continue using `specified::Color::CurrentColor` instead of this import, but fine either way.

Reply:
caused changed it back to `use crate::values::specified::Color;`
```console
munish@Zoro:~/stylo$ cargo build --features servo
   Compiling style v0.0.1 (/home/munish/stylo/style)
error[E0433]: failed to resolve: use of undeclared type `Color`
   --> /home/munish/stylo/target/debug/build/style-6b3c86fc1eff8679/out/shorthands/text.rs:177:48
    |
177 | ...n_color == Color::CurrentColor
    |               ^^^^^ use of undeclared type `Color`
    |
help: consider importing one of these items
    |
11  +         use crate::color::parsing::Color;
    |
11  +         use crate::properties::computed::Color;
    |
11  +         use crate::values::animated::color::Color;
    |
11  +         use crate::values::generics::color::Color;
    |
      and 1 other candidate

For more information about this error, try `rustc --explain E0433`.
error: could not compile `style` (lib) due to 1 previous error
```

Reply:
I faced an error regarding imports. As it's an issue I changed it to `use crate::values::specified::Color;`

Reply:
I meant that below it wasn't necessary to change `*self.text_decoration_color == specified::Color::CurrentColor` into `*self.text_decoration_color == Color::CurrentColor`. But no problem, it's also fine as is.